.run/PostHog.run.xml

```diff
@@ -4,15 +4,16 @@
     <option name="INTERPRETER_OPTIONS" value="" />
     <option name="PARENT_ENVS" value="true" />
     <envs>
-      <env name="PYTHONUNBUFFERED" value="1" />
+      <env name="BILLING_SERVICE_URL" value="https://billing.dev.posthog.dev" />
       <env name="CLICKHOUSE_SECURE" value="False" />
       <env name="DATABASE_URL" value="postgres://posthog:posthog@localhost:5432/posthog" />
       <env name="DEBUG" value="1" />
       <env name="DJANGO_SETTINGS_MODULE" value="posthog.settings" />
       <env name="KAFKA_URL" value="kafka://localhost" />
+      <env name="KEA_VERBOSE_LOGGING" value="true" />
```

Comment:
I don't think I'd want this on all the time, especially if just for a few people... 🤔